lib/matplotlib/animation.py

```diff
@@ -1392,26 +1391,20 @@ class TimedAnimation(Animation):
 
     Parameters
     ----------
-    fig : `~matplotlib.figure.Figure`
-       The figure object that is used to get draw, resize, and any
-       other needed events.
-
-    interval : number, optional
-       Delay between frames in milliseconds.  Defaults to 200.
-
-    repeat_delay : number, optional
+    fig : `~.Figure`
+        The figure object that is used to get draw, resize, and any
+        other needed events.
+    interval : int, default: 200
+        Delay between frames in milliseconds.
+    repeat_delay : int, default: None
         If the animation in repeated, adds a delay in milliseconds
```

Comment:
```suggestion
         If the animation is repeated, adds a delay in milliseconds
```
But while we're at it, could be written as
> The delay between subsequent runs of the animation, if it is repeated.

Reply:
agreed (modulo wording)